lib/t_web/channels/feed_channel.ex

```diff
@@ -83,39 +71,12 @@ defmodule TWeb.FeedChannel do
     {:reply, {:ok, %{"feed" => render_feed(feed, screen_width), "cursor" => cursor}}, socket}
   end
 
-  def handle_in("archived-matches", _params, socket) do
-    %{current_user: user, screen_width: screen_width} = socket.assigns
-
-    archived_matches =
-      user.id
-      |> Matches.list_archived_matches()
-      |> render_matches(screen_width)
-
-    {:reply, {:ok, %{"archived_matches" => archived_matches}}, socket}
-  end
-
-  def handle_in("archive-match", %{"match_id" => match_id}, socket) do
-    Matches.mark_match_archived(match_id, me_id(socket))
-    {:reply, :ok, socket}
```

Comment:
Should this reply with an alert? Or are all users on 6.0.0 already?

Reply:
@ruslandoga this PR will be merged when users are on 6.0.0

Reply:
I'd prefer merging it before that